src/doc/reference.md

```diff
@@ -845,6 +845,20 @@ extern crate std; // equivalent to: extern crate std as std;
 extern crate std as ruststd; // linking to 'std' under another name
 ```
 
+When naming Rust crates, hyphens are disallowed. However, Cargo packages may
+make use of them. In such case, when `cargo.toml` doesn't specify a crate name,
+Cargo will transparently replace `-` with `_` (Refer to [RFC 940] for more
+details).
+
+Here is an example:
+
+```{.ignore}
+// Importing the Cargo package hello-world/
+extern crate hello_world; // The crate name replaced the hyphen
```

Comment:
* trailing `/` looks suspicious?
* did you mean "The underscore replaced the hyphen"?